libraries/GCS_MAVLink/GCS.h

```diff
@@ -354,6 +354,9 @@ class GCS_MAVLINK
     void send_rc_channels() const;
     void send_rc_channels_raw() const;
     void send_raw_imu();
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
```

Comment:
Do you not want to link this to HAL_INS_HIGHRES_SAMPLE or am I misunderstanding?

Reply:
probably a good idea but not a requirement since you could be wanting baro or mag..